addons/partner_autocomplete/models/res_partner.py

```diff
@@ -124,12 +126,21 @@ def enrich_company(self, company_domain, partner_gid, vat, timeout=15):
 
     @api.model
     def read_by_vat(self, vat, timeout=15):
+        check_vies_res = check_vies(vat)
+        del check_vies_res['requestDate']
```

Comment:
Prefer `pop` over `del`:
```suggestion
        check_vies_res.pop('requestDate')
```